packages/dialtone-tokens/tokens/base/dark.json

```diff
@@ -26,23 +26,23 @@
         "type": "color"
       },
       "600": {
-        "value": "#B0B0B0",
+        "value": "#b0b0b0",
```

Comment:
Do you know why most of the tokens changed from capital letters to lowercase? this is making a lot of noise in the PR making it harder to spot mistakes

Reply:
No idea. @jcarter-dp is there something about your settings for Tokens Studio, or perhaps Figma, that converts them to lowercase? 

Quick fix: open branch in Text Editor and select those lines and transform them to uppercase. That will make it easier to PR review them. 

https://github.com/user-attachments/assets/2bc6ef18-b3ba-4e0f-a7b3-44f4ae168f39